week2/2-website/css/style.css

```diff
@@ -5,6 +5,194 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+
+#logo {
+    display: flex;
+    justify-content: center;
+    width: 43vw;
+}
+#header-img {
+    width: 40px;
+    height: 40px;  
+}
+nav {
+    width: 57vw;
+    display: flex;
+    justify-content: flex-start;
+}
+nav ul li {
+    display: inline-block; 
+    padding: 10px 15px 10px 15px;
+    font-weight: 500;
+    
+}
+nav ul li:first-of-type {
```

Comment:
This is good to use `first-of-type` however - in this case - I would say it is bold because it is the active page so it won't always be `first-of-type`. Try using an `active` class instead